contollers/articles_controller.js

```diff
@@ -43,4 +43,13 @@ exports.patchArticle = (req, res, next) => {
         res.status(200).send({article})
     })
     .catch(next)
+}
+
+exports.deleteComment = (req, res, next) => {
+    const commentID = req.params.comment_id
+    removeComment(commentID)
+    .then(() => {
+        res.status(204).send({})
```

Comment:
I think we should just not send anything back - res.sendStatus(204) or res.status(204).send() would both be fine :)